prep-javascript-dom-creation/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript DOM Creation</title>
+</head>
+
+<body>
+  <div id="container">
+
+  </div>
+  <script>
+    const $container = document.querySelector('#container');
+    console.log($container);
+
+    let $heading = document.createElement('h1');
+    console.log($heading);
+
+    $heading = document.createTextNode('Hello World!');
```

Comment:
Here you'll want to use a separate variable for the text node. then you can append the text node to the heading element, and finally the heading element to the container element

Reply:
<img width="2359" alt="Screen Shot 2022-01-22 at 9 47 44 AM" src="https://user-images.githubusercontent.com/20695764/150649803-cd6b26f9-0bc7-433b-bc78-46d373bec576.png">